src/lib/components/organisms/coalitions-tracker/index.jsx

```diff
@@ -74,24 +74,24 @@ export function CoalitionsTracker({
     <div ref={wrapperRef} className={styles.coalitionsWrapper}>
       <div className={styles.coalitionsContainer}>{parsedLists.map(renderCoalition)}</div>
       <div
+        className={styles.thresholdDot}
         style={{
```

Comment:
small comment - but since you have a new class here - want to add some of the inline styles into the class - things like this to clean it up: `position: 'absolute',`